### DISCUSSION TITLE
Missing 'profile' Folder mentioned in Backup Documentation

### DISCUSSION THREAD

**Discussion by chem-william** [immich-app/immich#4214] [Q&A] [answered]
In the [documentation](https://documentation.immich.app/docs/administration/backup-and-restore#filesystem) it is mentioned that the only three folders that needs to be backed up is the following:
1.`UPLOAD_LOCATION/library`
2. `UPLOAD_LOCATION/upload`
3. `UPLOAD_LOCATION/profile`

I only see `library/` and `upload/` in my `UPLOAD_LOCATION`. Have I done something wrong or is the documentation out-of-date? Or something else entirely?

**Comment by bo0tzz** [comment, marked as answer]
The profile folder only gets created if you set a profile picture for a user.

**Comment by chem-william** [comment]
Wonderful! Thanks for the quick answer
